sanic/blueprints.py

```diff
@@ -82,15 +83,18 @@ def decorator(handler):
             return handler
         return decorator
 
-    def add_route(self, handler, uri, methods=None, host=None):
+    def add_route(self, handler, uri, methods=frozenset({'GET'}), host=None):
         """
         Creates a blueprint route from a function.
-        :param handler: Function to handle uri request.
+        :param handler: Function or class instance to handle uri request.
```

Comment:
Should this be more explicit? For example "Function for handling uri requests. Accepts function, or class instance with a `view_class` method."